shell/client/shell.html

```diff
@@ -1062,6 +1038,37 @@ <h4 class="share-with-others">Share with others</h4>
         <p><button id="admin-settings-send-test">Send</button></p>
       </p>
     {{/if}}
+    {{#if isFeatureKeyValid}}
+      <h4>LDAP login</h4>
+      <p class="forwork-note">Sandstorm for Work feature</p>
+
+      <p><label>
+        <input type="checkbox" name="ldapLogin" value="value" checked={{ldapEnabled}}> Enable LDAP Login
+      </label><br>
+        <label>LDAP Url: <input id="ldapUrl" type="text" name="ldapUrl" value="{{ldapUrl}}"></label>
+        <label>LDAP DN Pattern ($USERNAME will be replaced): <input id="ldapDnPattern" type="text" name="ldapDnPattern" value="{{ldapDnPattern}}"></label>
+        <label>LDAP Base Search Dn: <input id="ldapBase" type="text" name="ldapBase" value="{{ldapBase}}"></label>
+        <label>LDAP Name Field: <input id="ldapNameField" type="text" name="ldapNameField" value="{{ldapNameField}}"></label></p>
+
+      <h4>SAML login</h4>
+      <p class="forwork-note">Sandstorm for Work feature</p>
+
+      <p>Coming soon!</p>
+
+      <h4>Define oragnization</h4>
+      <p class="forwork-note">Sandstorm for Work feature</p>
+
+      <p>These users will be considered part of your organization. They will automatically be
+        able to log in, install apps, and create grains.</p>
+
+      <p>
+        <label><input type="checkbox" name="isOrganizationLdap" value="value" checked={{isOrganizationLdap}}> Users authenticated via LDAP.</label><br>
+        <label><input type="checkbox" name="isOrganizationGoogle" value="value" checked={{isOrganizationGoogle}}> Users authenticated via Google Apps for Work domain:</label> <input type="text" name="organizationGoogle" value="{{organizationGoogle}}" placeholder="example.com">
+        <label><input type="checkbox" name="isOrganizationEmail" value="value" checked={{isOrganizationEmail}}>Users authenticated via passwordless email login with addresses at domain:</label> <input type="text" name="organizationEmail" value="{{organizationEmail}}" placeholder="example.com"></p>
+    {{else}}
+      <p>To get support for LDAP, SAML, and organization management, enable
+        <a href="{{featuresPath}}">Sandstorm for Work</a>.</p>
```

Comment:
I'd slightly prefer using iron router's linkto helper (https://github.com/iron-meteor/iron-router/blob/devel/Guide.md#linkto), but not a huge deal.